packages/sdks/src/components/blocks/blocks-wrapper.lite.tsx

```diff
@@ -50,10 +50,13 @@ export default function BlocksWrapper(props: BlocksWrapperProps) {
       if (!props.path) {
         return undefined;
       }
+      const thisPrefix = 'this.';
       const pathPrefix = 'component.options.';
-      return props.path.startsWith(pathPrefix)
-        ? props.path
-        : `${pathPrefix}${props.path || ''}`;
+      return props.path.startsWith(thisPrefix)
+        ? props.path.replace(thisPrefix, '')
+        : props.path.startsWith(pathPrefix)
+          ? props.path
+          : `${pathPrefix}${props.path || ''}`;
```

Comment:
Thanks for the catch!

added children support for `<Blocks />` for this
> I'm unable to drag'n'drop successfully into an empty Variant Container block

we need to render them as `<Blocks />` because:
1. we need the BlocksWrapper to render so that hydration matches with the other variants
2. this will allow us to drag and drop - will enable empty blocks